### DISCUSSION TITLE
Grid max width for cards and inputs

### DISCUSSION THREAD

**Discussion by lgalloherran** [twilio-labs/paste#2837] [Q&A] [answered]
Hi Team, 
Do we have a max width for components to be displayed on a page? 
My engineering team is asking this because the components are currently being adapted to max width of the page and they seem too wide.

**Comment by ghost** [comment]
Hi @lgalloherran thanks for the discussion here. I've asked the product design side of our team to weigh in here

**Comment by TheSisb** [comment, marked as answer]
Hi @lgalloherran,

Most components in Paste are set to 100% width so they can responsively fit inside any container. This gives developers and designers freedom to use components in a range of UIs without strange behaviors. It is currently up to teams to figure out what the best design and widths to use in their apps, for their specific responsive breakpoints; some apps are fluid and others are fixed width. That said, we are going to be working on a high-level guidance for layouts and spacing in 2023, as part of our work on One Twilio. Stay tuned for more info!

If this is a blocker for your team and you want more personalized feedback, consider signing up for our office hours.

**Reply by lgalloherran** [reply]
Hey Shadi, thanks for the info, I'll report this to my engineering team.
